docs/internals/optimizer.rst

```diff
@@ -1018,6 +1058,12 @@ a structural level:
 
 This component uses the DataflowAnalyzer.
 
+LiteralRematerialiser should be run before this step.
+
+Prerequisites: Disambiguator.
+
+Important: Can only be used on EVM code.
```

Comment:
Not sure about usefulness of this either now that we only have an EVM backend, but since other steps already list this kind of info (e.g. `Important: Introduces EVM opcodes and thus can only be used on EVM code for now` for ControlFlowSimplifier) I added it where it was available for now.

Reply:
The code only has that, because we haven't removed all of it when removing the EWASM parts :-).

Reply:
Remarks about EVM removed from the docs and also from the docstrings.